src/index.js

```diff
@@ -23,25 +35,25 @@ const _createFabricRefCode = (refIdentifier, typeIdentifier, propsIdentifier) =>
   ]; 
   const isTurboModuleEnabled = global.RN$Bridgeless || global.__turboModuleProxy != null
   if (isTurboModuleEnabled && Platform.OS === 'ios') {
-   if (${typeIdentifier}.$$typeof && (${typeIdentifier}.$$typeof.toString() === 'Symbol(react.forward_ref)' || ${typeIdentifier}.$$typeof.toString() === 'Symbol(react.element)' || ${typeIdentifier}.$$typeof.toString() === 'Symbol(react.transitional.element)')) {
-      if (${propsIdentifier}) {
-        const propContainsFSAttribute = SUPPORTED_FS_ATTRIBUTES.some(fsAttribute => {
-          if (!!props[fsAttribute]) {
-            if (fsAttribute === 'fsAttribute') {
-              return typeof props[fsAttribute] === 'object';
-            } else {
-              return typeof props[fsAttribute] === 'string';
-            }
+  if (isReact19Plus() || (${typeIdentifier}.$$typeof && (${typeIdentifier}.$$typeof.toString() === 'Symbol(react.forward_ref)' || ${typeIdentifier}.$$typeof.toString() === 'Symbol(react.element)' || ${typeIdentifier}.$$typeof.toString() === 'Symbol(react.transitional.element)'))) {
+    if (${propsIdentifier}) {
+      const propContainsFSAttribute = SUPPORTED_FS_ATTRIBUTES.some(fsAttribute => {
+        if (!!${propsIdentifier}[fsAttribute]) {
+          if (fsAttribute === 'fsAttribute') {
+            return typeof ${propsIdentifier}[fsAttribute] === 'object';
```

Comment:
Bugfix: `props[fsAttribute]` -> `${propsIdentifier}[fsAttribute]`. We cannot assume that the variable will always be named `props`. We need to dynamically fill this. See the sample `output.js`s to see it in action.